docs(vscode): outline more that global teardown needs to be executed manually (#33221)
microsoft/playwright: docs/src/getting-started-vscode-js.md

## Global Setup

> **Global setup** tests are run when you execute your first test. This runs only once and is useful for setting up a database or starting a server. You can manually run a **global setup** test by clicking the `Run global setup` option from the **Setup** section in the Playwright sidebar. **Global teardown** tests do not run by default; you need to manually initiate them by clicking the `Run global teardown` option.

Reviewer: ```suggestion
**Global setup** runs when you execute your first test. It runs only once and is useful for setting up a database or starting a server. You can manually run **global setup** by clicking the `Run global setup` option from the **Setup** section in the Playwright sidebar. **Global teardown** does not run by default; you need to manually initiate it by clicking the `Run global teardown` option.
```
